README.md

```diff
@@ -227,6 +227,10 @@ pod 'YapDatabaseExtensions/Functional’
 
 API documentation is available on [CocoaDocs.org](http://cocoadocs.org/docsets/YapDatabaseExtensions).
 
+## Developing
+
+To start working in this repository’s `YapDatabaseExtensions.xcodeproj`, you’ll need to use [Carthage](https://github.com/Carthage/Carthage#installing-carthage) to download & build the project’s dependencies, with the commands `carthage checkout` and `carthage build`.
```

Comment:
Hm, [this helpful blog post from Thoughtbots](https://robots.thoughtbot.com/creating-your-first-ios-framework) indicates that the following command should be used:

```
carthage update --platform iOS --use-submodules --no-use-binaries
```

@danthorpe I haven't used Carthage much before. How have you been achieving this on your machine?